ListAndBookTask/BookTasks.java

```diff
@@ -0,0 +1,36 @@
+package ListAndBookTask;
+
+import Utils.BookLanguages;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class BookTasks {
+
+    public Comparator<Map<String, String>> mapComparator = (m1, m2) -> m1.get("author").compareTo(m2.get("author"));
+
+    public void BookExample() {
+        ListOfBooks list = new ListOfBooks();
+
+        Map<String, String> spanishBook = list.getBooks().stream()
+                .filter(map -> map.get("language").equals(BookLanguages.SPANISH.name()))
+                .findFirst().orElseThrow(NoSuchElementException::new);
+
+        System.out.println("First Spanish Book from the list: \n" + spanishBook);
+
+        Collections.sort(list.getBooks(), mapComparator);
```

Comment:
According to the task, books should be ordered by author name in ascending order.